static/panes/editor.js

```diff
@@ -674,7 +674,7 @@ Editor.prototype.initEditorActions = function () {
     });
 
     this.editor.addAction({
-        id: 'clang-format',
+        id: 'format',
```

Comment:
I recently learned that Monaco itself has some built-in formatting (Check #2770). I wonder if there's a way to mix both here instead of having a separate action. But maybe that's for a second PR

Reply:
Looks like the monaco stuff is a bit of a non-trivial change so I think it's easier to leave that for another PR.